src/main/java/com/getindata/connectors/http/internal/table/lookup/querycreators/QueryFormatAwareConfiguration.java

```diff
@@ -24,6 +23,7 @@ class QueryFormatAwareConfiguration extends Configuration {
 
     QueryFormatAwareConfiguration(String queryFormatName, Configuration other) {
         super(other);
+        this.classLoader = other.getClass().getClassLoader();
```

Comment:
you have [removed](https://github.com/getindata/flink-http-connector/pull/68/commits/6416e18866f3145b1be81df3f7194f0d693d73c3) previously added `classLoader` field but this line left -> compilation error.

Reply:
sorry - I am not sure how this got through :-( , Removing